charts/kube-green/templates/deployment.yaml

```
@@ -0,0 +1,118 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: kube-green
+    control-plane: controller-manager
+    {{- include "kube-green.labels" . | nindent 4 }}
+  name: kube-green-controller-manager
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
```

```suggestion
  replicas: {{ .Values.replicaCount }}
```
I would make it configurable through values for HA setup

---

I think the controller cannot be used with more replicas. I'm correct @davidebianchi ?

---

Yes, you are correct. It should be tested the support, at the moment I think it's ok to not make it configurable